.github/ISSUE_TEMPLATE/midpoint-review.md

```diff
@@ -61,11 +61,6 @@ Provide links or documents for the following:
 
 ### Content artifacts 
 
-- [ ] Content brief (if applicable) and your product’s content source of truth 
-<!-- 
-This may include mockups, prototypes, Github files, Word document or outline, staging link, or preview link. 
--->
-
 - [name of content of truth, for example, "Invision Prototypes"](link here)
```

Comment:
I think this line should also be removed since it is a sub-bullet of the "Content brief (if applicable) and your product's content source of truth" that we are removing.
What do you think?

Reply:
@chaseakirby ok, I did not realize it went with content brief, it looked weird to me lol I will remove and then publish